desktop/main/startApp.ts

```diff
@@ -150,6 +151,9 @@ const startApp = (): AppStore => {
   const $runNodeBeforeLogin = new $.BehaviorSubject<boolean>(
     StoreService.get(IS_AUTO_START_ENABLED) || startNodeAfterUpdate
   );
+  const $osPlatform = new $.BehaviorSubject({
+    osPlatform: os.platform(),
+  }).asObservable();
```

Comment:
Why do we need this over-complication?
This subject won't ever change.
Also, I don't understand the reason for having extra record inside :)